source/modules/MEI.xml

```diff
@@ -3022,8 +3064,20 @@
             </valItem>
           -->
         <valItem ident="tab">
-          <desc xml:lang="en">Tablature notation.</desc>
+          <desc xml:lang="en">Generic tablature notation.</desc>
```

Comment:
Perhaps "Unspecified" or "Modern"? "Generic" doesn't really seem like the right word here.

Reply:
We have reverted to "Tablature notation" to prevent confusion